app/components/PureSidebar.tsx

```diff
@@ -88,6 +89,19 @@ export const PureSidebar = ({ isOpen, setIsOpen, isTransition, setIsTransition }
             </Button>
           )}
           </div>
+          <div className="flex items-center w-full pt-10" >
+            <Avatar className="w-12 h-12" >
+              <AvatarFallback className="bg-gray-800 text-white font-semibold" >SW</AvatarFallback>
+            </Avatar>
+            <div className={"px-4 transition-all duration-500 whitespace-nowrap"}>
+              <h2 className="text-gray-900 font-semibold leading-6 text-lg">
```

Comment:
```suggestion
 <h2 className="text-gray-900 font-semibold overflow-hidden overflow-ellipsis">
```
1. Why `leading-6`? Where did you take this from?
2. Same for `text-lg`?!?